sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -0,0 +1,340 @@
+## API Report File for "@azure/attestation"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+
+// @public
+export type AttestationAttestOpenEnclaveResponse = AttestationResponse & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: AttestationResponse;
+    };
+};
+
+// @public
+export type AttestationAttestSgxEnclaveResponse = AttestationResponse & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: AttestationResponse;
+    };
+};
+
+// @public
+export type AttestationAttestTpmResponse = TpmAttestationResponse & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: TpmAttestationResponse;
+    };
+};
+
+// @public
+export interface AttestationCertificateManagementBody {
+    policyCertificate?: JsonWebKey;
+}
+
+// @public (undocumented)
+export class AttestationClient extends AttestationClientContext {
+    constructor(credentials: coreHttp.TokenCredential | coreHttp.ServiceClientCredentials, instanceUrl: string, options?: AttestationClientOptionalParams);
+    // Warning: (ae-forgotten-export) The symbol "Attestation" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    attestation: Attestation;
+    // Warning: (ae-forgotten-export) The symbol "MetadataConfiguration" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    metadataConfiguration: MetadataConfiguration;
+    // Warning: (ae-forgotten-export) The symbol "Policy" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    policy: Policy;
+    // Warning: (ae-forgotten-export) The symbol "PolicyCertificates" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    policyCertificates: PolicyCertificates;
+    // Warning: (ae-forgotten-export) The symbol "SigningCertificates" needs to be exported by the entry point index.d.ts
```

Comment:
@joheredi why code gen could not export those as well?

Reply:
Can we add these manually for now? I think not having these exported could cause some compilation errors for end-users that try to use these fields.

Reply:
I manually added exports from `operations.ts`.